pnpm-lock.yaml

```diff
@@ -3077,19 +3077,19 @@ importers:
       '@types/jest': 29.5.3
       '@types/jest-environment-puppeteer': link:../../packages/test/types_jest-environment-puppeteer
       '@types/lodash.isequal': 4.5.8
-      '@types/node': 18.19.1
       '@types/node-fetch': 2.6.11
       '@types/react': 17.0.71
       '@types/react-dom': 17.0.25
       '@types/valid-url': 1.0.7
       clean-webpack-plugin: 4.0.0_webpack@5.89.0
+      copyfiles: 2.4.1
       cross-env: 7.0.3
       eslint: 8.55.0
       eslint-config-prettier: 9.0.0_eslint@8.55.0
       expect-puppeteer: 9.0.2
       good-fences: 1.2.0
       html-webpack-plugin: 5.5.3_webpack@5.89.0
-      jest: 29.7.0_@types+node@18.19.1
+      jest: 29.7.0
```

Comment:
This is a nice change to see. I guess we could look to see how many other jest+@types/node dependees there are.